W/wolfSSL/build_tarballs.jl

```diff
@@ -16,13 +16,18 @@ cd $WORKSPACE/srcdir/wolfssl*
 
 mkdir build && cd build
 
+if [[ "${target}" == *-apple-darwin* ]]; then
+    CFLAGS="-mmacosx-version-min=10.14"
+fi
```

Comment:
1. You're matching also aarch64-apple-darwin, which doesn't have the same problem (we use the 11.0 SDK there)
2. it isn't sufficient to pretend to have the 10.14 SDK, you actually need to use it as suggested in https://github.com/JuliaPackaging/Yggdrasil/pull/9578#issuecomment-2399871955 :slightly_smiling_face:

Side node, `export MACOSX_DEPLOYMENT_TARGET=10.14` is a nicer way to accomplish this, since our compiler wrappers read the `MACOSX_DEPLOYMENT_TARGET` env var, the way you do it the compiler gets duplicate and conflicting `-mmacosx-version-min` options.

Reply:
Thanks! Did that and it worked. Any chance we can merge this in without the Windows build?

Reply:
Trying something for the windows build now.